deploy/yamls/zfssnapshot-crd.yaml

```diff
@@ -104,10 +115,13 @@ spec:
                 volume should be cloned
               type: string
             thinProvision:
-              description: Thinprovision specifies if we should thin provisioned the
-                volume or not
+              description: ThinProvision describes whether we have reserved the space
```

Comment:
what is the default value, if not specified by user? 

"yes" => volume can be provisioned even if the ZPOOL doesn't have the required capacity. 
"no"  => volume can be provisioned only if the ZPOOL has enough capacity and capacity required by volume can be reserved.

Reply:
Btw, could not comment on the unchanged lines above. Please check the description for the `properties` field.  It says volumeinfo. Also as these are snapshots, it is probably worth mentioning how values are populated.

also it will be good to indicate if a given attribute is editable or only set during the creation time.

Reply:
n_: compression description has a grammar error.